deploy/tower-operator.yaml

```diff
@@ -113,14 +113,14 @@ spec:
             - /usr/local/bin/ao-logs
             - /tmp/ansible-operator/runner
             - stdout
-          image: "matburt/tower-operator:latest"
+          image: "ansible/tower-operator:latest"
```

Comment:
(Just as a note, mostly for anyone else reading this in the immediate future, https://hub.docker.com/u/ansible/tower-operator doesn't yet exist.)